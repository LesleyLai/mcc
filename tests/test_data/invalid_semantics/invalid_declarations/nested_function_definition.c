int main(void)
{
  /* Nested function definitions are not permitted */
  int foo(void)
  {
    return 1;
  }
  return foo();
}
