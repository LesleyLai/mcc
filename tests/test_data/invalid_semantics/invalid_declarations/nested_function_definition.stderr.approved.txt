{{filename}}:3:7: Error: nested function definition is not permitted
3 |   int foo(void)
4 |   {
5 |     return 1;
6 |   }

